Research/HumicAcidData_Yingda.py

```diff
@@ -12,20 +12,15 @@
 import pandas as pd
 from matplotlib import pyplot as plt
 
-try:
-    from AguaClara_design.units import unit_registry as u
-    from AguaClara_design import floc_model as floc
-except ModuleNotFoundError:
-    try:
-        import floc_model as floc
-        from units import unit_registry as u
-    except ModuleNotFoundError:
-        import sys, os
-        GitPath = os.path.join('~', 'Documents', 'GitHub')
-        myGitHubdir=os.path.expanduser(GitPath)
-        sys.path.append(myGitHubdir)
-        from AguaClara_design.units import unit_registry as u
-        from AguaClara_design import floc_model as floc
+from aide_design import floc_model as floc
```

Comment:
I don't understand why this research is in here. Should it be put into its own repo?